File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] strArr = new String[3];

        for (int i = 0; i < 3; i++) {
            strArr[i] = br.readLine();
        }

        int ans = 0;
        for (int i = 0; i < strArr.length; i++) {
            String cur = strArr[i];

            if (!(cur.equals("Fizz") || cur.equals("Buzz") || cur.equals("FizzBuzz"))) {
                ans = Integer.parseInt(cur) + (3 - i);
                break;
            }
        }

        if (ans % 3 == 0 && ans % 5 == 0) {
            System.out.println("FizzBuzz");
        } else if (ans % 5 == 0) {
            System.out.println("Buzz");
        } else if (ans % 3 == 0) {
            System.out.println("Fizz");
        } else {
            System.out.println(ans);
        }
    }
}

File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.kt
import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader

@Throws(IOException::class)
fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val inputs = List(3) { readLine() }

    var ans = 0
    for ((i, cur) in inputs.withIndex()) {
        if (cur != "Fizz" && cur != "Buzz" && cur != "FizzBuzz") {
            ans = cur.toInt() + (3 - i)
            break
        }
    }

    val output = when {
        ans % 15 == 0 -> "FizzBuzz"
        ans % 5 == 0 -> "Buzz"
        ans % 3 == 0 -> "Fizz"
        else -> ans
    }

    print(output)
}